lib/msf/core/exploit/sqli/utils/utils.rb

```diff
@@ -0,0 +1,30 @@
+module Msf::Exploit::SQLi
+
+  #
+  # This module provides utilities used by the SQL injection library, none of the utilities are specific to SQL injection
+  #
+  module Utils
```

Comment:
```suggestion
module Msf::Exploit::SQLi::Utils

  #
  # This module provides utilities used by the SQL injection library, none of the utilities are specific to SQL injection
  #
  module Common
```

Consider rename path `lib/msf/core/exploit/sqli/utils/common.rb` and adjusting module name to match.